homeassistant/components/sharkiq/update_coordinator.py

```diff
@@ -74,7 +74,7 @@ async def _async_update_data(self) -> bool:
             SharkIqNotAuthedError,
             SharkIqAuthExpiringError,
         ) as err:
-            _LOGGER.exception("Bad auth state")
+            _LOGGER.debug("Bad auth state.  Attempting re-auth.", exc_info=err)
```

Comment:
We don't end logging messages with period.
```suggestion
            _LOGGER.debug("Bad auth state.  Attempting re-auth", exc_info=err)
```